### PR TITLE
Fix issue #384: There's no Git for Windows 2.x

### LINKED ISSUE #384: There's no Git for Windows 2.x
> On Windows systems, Git looks for the .gitconfig file in the $HOME directory (C:\Users\$USER for most people). It also still looks for /etc/gitconfig, although it’s relative to the MSys root, which is wherever you decide to install Git on your Windows system when you run the installer. However, **if you are using Git for Windows 2.x**, it is C:\Documents and Settings\All Users\Application Data\Git\config on Windows XP, and it is C:\ProgramData\Git\config on Windows Vista and newer.

I'm pretty sure there's no Git for Windows 2.x, nor Windows 3.x, Windows 95, etc..
I assume the author was going for:
> if you are using Git 2.x for Windows

In this case, since the book is about Git 2.0 onwards, a better way would be: 
> On Windows systems, Git looks for the .gitconfig file in the $HOME directory (C:\Users\$USER for most people). *It also still looks for C:\Documents and Settings\All Users\Application Data\Git\config on Windows XP, and C:\ProgramData\Git\config on Windows Vista and newer.*

### PR DESCRIPTION
**Pull request by YueLinHo**
Fixes #384 

Use "version 2.x of Git for Windows"

### CODE REVIEW

File: book/01-introduction/sections/first-time-setup.asc [hunk 1]
```diff
@@ -18,7 +18,7 @@ Each level overrides values in the previous level, so values in `.git/config` tr
 
 On Windows systems, Git looks for the `.gitconfig` file in the `$HOME` directory (`C:\Users\$USER` for most people).
 It also still looks for `/etc/gitconfig`, although it's relative to the MSys root, which is wherever you decide to install Git on your Windows system when you run the installer.
-If you are using Git for Windows 2.x or later, there is also a system-level config file at
+If you are using version 2.x of Git for Windows or later, there is also a system-level config file at
```
Comment (ben) [inline book/01-introduction/sections/first-time-setup.asc]: Let's change this a tad: "version 2.x or later of Git for Windows"

Comment (YueLinHo) [inline book/01-introduction/sections/first-time-setup.asc]: Sure.

**Comment by YueLinHo** [comment]
Updated!

**Comment by ben** [comment]
:+1:

**Comment by YueLinHo** [comment]
@ben thanks. :smile:

**Pull request merged**